.github/workflows/build_docs.yaml

```
@@ -0,0 +1,34 @@
+name: Build Documentation
+
+on:
+  pull_request:
+    paths:
+      - 'docs/**'
+    types:
+      - closed
```

Shouldn't we check if it was merged into main?

```suggestion
  pull_request:
     paths:
       - 'docs/**'
     types:
       - closed
    branches:
      - main
```

---

Yes, good point.

---

Addressed in commit 18f1849